puppetboard/app.py

```diff
@@ -362,16 +384,22 @@ def reports(env, page):
     """
     check_env(env)
 
+    if env == '*':
+        reports_query = None
+        total_query = '["extract", [["function", "count"]]]'
```

Comment:
This patch that I suggested will be compatible with PuppetDB 3.0 and 3.1. No need to break it.